campaigns/rustreach/how-I-came-to-rust.md

```diff
@@ -0,0 +1,58 @@
+---
+title: How I came to Rust
+---
+
+# How I came to Rust
+
+The #rustreach program asked people to write down how they came to Rust and I'm happy to do that. Entering a community or a space is always a moment of orientation process and I hope I can give some insight into that.
+
+My path is highly individual, but yours will be, too. Own that. It's great.
+
```

Comment:
Might you have an image or two that you could include? In my experience, including at least one photo/visual in a blog posts can really help! It gives a visual hook when the post is shared on social media, for example, and just helps make it look more appealing in general. Perhaps a photo of you at one of your earlier Rust events, for example?

Reply:
I find visuals distracting, so I avoid them on my web spaces, too. We also have no pictures of the Hack & Learn.